web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -112,8 +118,20 @@ export const SidebarCardsList = (props: Props): ReactElement => {
 
   return (
     <>
-      <h2 className="h1-styling margin-top-0">{Config.dashboardDefaults.sidebarHeading}</h2>
-      <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+      {isDesktopAndUp ? (
+        <>
+          <h2 className="margin-top-0 text-normal-override">
+            {" "}
+            {Config.dashboardDefaults.sidebarHeading}
+            <span data-testid="for-you-counter" className="margin-left-05 text-base">
+              ({getContentWithCardCount(business, props.certifications, props.fundings)})
+            </span>
+          </h2>
+          <hr className="margin-top-2 margin-bottom-3 bg-cool-lighter" aria-hidden={true} />
+        </>
+      ) : (
+        <></>
```

Comment:
[pebble] generally [we don't use the ternary style](https://njcio.slack.com/archives/C04J8MTJNS2/p1692719694434699) like `condition ? <Component/> : <></>` with an empty fragment, but prefer `&&` instead.

Reply:
totally agree..